packages/vscode-extension/src/utils/envTreeUtils.ts

```diff
@@ -34,20 +37,13 @@ export async function getSubscriptionInfoFromEnv(
 }
 
 export async function getM365TenantFromEnv(env: string): Promise<string | undefined> {
-  let provisionResult: Record<string, any> | undefined;
-
-  try {
-    provisionResult = await getProvisionResultJson(env);
```

Comment:
This function seems not to be used any more. Could we delete it?

Reply:
The function is still used in other code blocks. I'll contact feature owner to update the logics.

Reply:
After offline discussion, please help to check the upgrade of `state.env.json` file later. `getProvisionResultJson` is still used in other codes.